### DISCUSSION TITLE
Uno Platform AutosuggestBox TextBoxStyle.. Width property is not working on ios!

### DISCUSSION THREAD

**Discussion by Daantje007** [unoplatform/uno#9075] [Q&A] [answered]
AutoSugestBox TextBox Width property not working, always same size on ios.
Uno-Platform Based on Uwp-Xamarin

**Comment by jeromelaban** [comment]
Thanks for the report. Could you provide a screenshot of what you're seeing, and what tells you that it's always the same size? If you can, try creating a simple repro sample would help.

**Reply by Daantje007** [reply]
Yes, horizontal stretch also does not seem to work.

**Comment by Daantje007** [comment]
![Screenshot 2022-06-17 225647](https://user-images.githubusercontent.com/95089394/174401445-02437e4c-78a6-4c7d-894c-df1f190c7b1b.png)

**Reply by Daantje007** [reply]
Also on ios it does not select the first header item by default (Focus) this is the case with UWP. but this is an other toppic

**Reply by Daantje007** [reply]
I think i found something.. you can set both MinWidths for ios.

**Reply by Daantje007** [reply]
So Horizontalaligniment  Stretch does not work, left/right does.
When you put two fixed MinWidths,  on autosugestbox and  textboxstyle it does change in ios. 
But i still cant do: binding to element x, path=width in style. 
Cant bind to parrent.witdh.
And xbind to function also does not seem to work.. 

So basicly it is not working like the control should. hmm i think
Tomorow i wil check on it further.. 

I am sorry that i cant fix some work arround and have to bother you with it J! GN!
Keep up the good work. XD

**Reply by Daantje007** [reply]
![WhatsApp Image 2022-06-18 at 9 09 59 AM](https://user-images.githubusercontent.com/95089394/174427327-91d87f28-ba0e-418b-9a4f-728e0959dba7.jpeg)

**Comment by Daantje007** [comment]
![WhatsApp Image 2022-06-18 at 9 09 59 AM](https://user-images.githubusercontent.com/95089394/174428515-bce8902e-a60e-4232-b788-448f89f36333.jpeg)

**Comment by Daantje007** [comment, marked as answer]
update: it work as should!
